### DISCUSSION TITLE
can't connect to any LXC container

### DISCUSSION THREAD

**Discussion by iCheater** [tteck/Proxmox#589] [Help Needed]
Hi! 
 - i can't connect to any LXC container via web gui
 - what is login/pass for LXC containers ? 
 
Here is facts:
- i'v install home assistant VM and connected to it with web-gui without any problems
- i've install Dashy with script ```bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/dashy-v4.sh)"```
- Instruction says:  ```Dashy Interface - IP:4000```
- I can see dashy is running with ip 192.168.1.94
![image](https://user-images.githubusercontent.com/13772028/193448376-6fcb028a-1072-41f3-88bf-7d3daf2a3775.png)
- i cant connect to any link https://192.168.1.94:4000/ or http://192.168.1.94:4000/
- nmap tells me there is no 4000 port opened

<details>
  <summary>console output</summary>
    
  ```console
root@proxmox:~# nmap -p- 192.168.1.94
Starting Nmap 7.80 ( https://nmap.org ) at 2022-10-02 12:50 MSK
Nmap scan report for 192.168.1.94
Host is up (0.0000060s latency).
Not shown: 65533 closed ports
PORT     STATE SERVICE
22/tcp   open  ssh
5355/tcp open  llmnr
MAC Address: E2:E4:E7:30:CF:55 (Unknown)

Nmap done: 1 IP address (1 host up) scanned in 0.84 seconds
  ```
  
</details>

- i have default network settings 
![image](https://user-images.githubusercontent.com/13772028/193449374-4d44f706-fae6-4062-82a7-a1140a523cac.png)
 
![image](https://user-images.githubusercontent.com/13772028/193449430-5a647378-e7de-4f1b-b10f-603e3e422031.png)
![image](https://user-images.githubusercontent.com/13772028/193449458-578b4769-e4fb-416f-b681-79828d53f31e.png)

**Comment by tteck** [comment]
> what is login/pass for LXC containers ?

You set that. If you use "Default settings" it automatically logs you in. If you want to use a password, go through "Advanced"

**Reply by iCheater** [reply]
oookey
- ill reinstall container
- but i am still curious what is default login password

**Comment by tteck** [comment]
> but i am still curious what is default login password

There is no default password.

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/324

**Reply by iCheater** [reply]
i've reinstalled dashy
- is it ok that there is a message "there is no internet connection?" 
![image](https://user-images.githubusercontent.com/13772028/193452687-f2c34159-f304-4c33-9d07-469574416f1b.png)
- i can set a password, but i can't set login, tried ```dashy``` and ```root```
![image](https://user-images.githubusercontent.com/13772028/193452837-e2f43662-37b7-4f49-8603-98204cc75d6c.png)
 - i can't access dashy web gui

**Reply by tteck** [reply]
When you get "Internet NOT connected" the script exits (does not finish) because there's no need to continue.
I'd suggest reading through https://github.com/tteck/Proxmox/issues/587 because you also have a variable error that needs to be fixed.

**Reply by iCheater** [reply]
well, 
- i'v removed ```line 345 my $node = $param->{node}; ``` in  ```nano /usr/share/perl5/PVE/API2/Disks/ZFS.pm ```
![image](https://user-images.githubusercontent.com/13772028/193455458-f539ead6-321b-4c2b-bd38-39c7ca5cd346.png)
- rebooted proxmox host
- reinstalled dashy and got same internet connection error

then,
- i've turn off ipv6 on my router
- rebooted router
- rebooted proxmox host
- reinstalled dashy - > same internet connection error

**Reply by tteck** [reply]
Then, you have other network issues.
Try using the Proxmox Shell (no ssh)

**Reply by iCheater** [reply]
- is there any way to diagnose? vm with home assistant works perfectly, containers not
- i've commented line 345 with double slash "//" and after reboot web-gui did not started, so i had to use ssh to remove the line

**Reply by tteck** [reply]
Try creating a LXC under the same node as the VM
